app/scripts/models/reliers/relier.js

```diff
@@ -106,6 +106,33 @@ define(function (require, exports, module) {
     },
 
     /**
+     * Set a value based on a value in window.location.search. Only updates
+     * model if parameter exists in window.location.search and passes validation.
+     *
+     * Throws Error mapped to `INVALID_PARAMETER` in Errors object.
+     *
+     * @param {String} sourceName - name of the search parameter
```

Comment:
You found an inconsistent convention - Can you lowercase the type in the jsdoc unless it's things like `Errors` where it's an internal type?